mq/publishers.go

```diff
@@ -108,6 +111,163 @@ func PublishSingleHostPeerUpdate(ctx context.Context, host *models.Host, deleted
 	return publish(host, fmt.Sprintf("peers/host/%s/%s", host.ID.String(), servercfg.GetServer()), data)
 }
 
+// FlushNetworkPeersToHost - sends all the peers in the network to the host.
+func FlushNetworkPeersToHost(host *models.Host, hNode *models.Node, networkNodes []models.Node) error {
```

Comment:
have you considered creating a client struct  instead of passing host and node all the time?
```
type Client struct {
  Host models.Host
  Node models.Node
}
```

Reply:
can be made into a struct if the parameters become four or more